lib/screens/take_picture_screen/take_picture_screen.dart

```diff
@@ -70,6 +72,9 @@ class _TakePictureScreenState extends State<TakePictureScreen> {
       // Attempt to take a picture and log where it's been saved.
       await _controller.takePicture(path);
 
+      var sc = SpeciesClassifier();
+      var species = await sc.getSpecies('assets/image.jpg');
```

Comment:
This species classification would preferably happen on the picture details screen, but if you don't have the time to do that we can just pass species as a parameter to picture details screen for the time being